components/single_image_view/single_image_view.tsx

```diff
@@ -144,7 +144,9 @@ export default class SingleImageView extends React.PureComponent<Props, State> {
         }
 
         const fileHeader = (
-            <div className='image-header'>
+            <div className={`image-header ${this.props.isEmbedVisible && 'active'}`}>
```

Comment:
could you please use the `classNames` (from the `classnames` package) helper here to conditionally assign a class?